src/extension/app/store/app.js

```diff
@@ -822,25 +837,15 @@ export class AppStore {
       })
       .then((json) => this.fireEvent(EXTERNAL_EVENTS.STATUS_FETCHED, json))
       .catch(({ message }) => {
-        this.status.error = message;
-        // TODO: Setup modals
-        // const modal = {
-        //   message: message.startsWith('error_') ? i18n(this, message) : [
-        //     i18n(this, 'error_status_fatal'),
-        //     'https://status.hlx.live/',
-        //   ],
-        //   sticky: true,
-        //   level: 0,
-        //   callback: () => {
-        //     // this error is fatal, hide and delete sidekick
```

Comment:
Do we need this callback?

Reply:
We don't. That was an implementation of the legacy error modal.